templates/_devcontainer/iobroker/_Dockerfile.ts

```diff
@@ -8,6 +8,9 @@ const templateFunction: TemplateFunction = answers => {
 	const template = `
 FROM iobroker/iobroker:latest
 RUN ln -s /opt/iobroker/node_modules/ /root/.node_modules
+# Needed to use git-ssh in devcontainer
+RUN apt-get update &&\\
```

Comment:
This is more of a personal preference, but I find multiline Docker commands easier to grasp if there is a space before the "newline" backspace, and the `&&` is in the next line before the 2nd command.

Reply:
done